rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketDefinitionController.java

```diff
@@ -171,6 +173,12 @@
         return response;
     }
 
+    private Workbasket remove_id(Workbasket importedWb) {
```

Comment:
Methoden sollten in CamelCase-Notation benannt werden. Warum weichst du hier davon ab?

Reply:
Danke für die Korrektur.
Hatte davor in einem Python-Projekt gearbeitet und da ist mir das untergekommen...